.github/workflows/lint-test.yml

```
@@ -0,0 +1,67 @@
+name: Lint & Test
+
+on: [push, pull_request]
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    name: flake8
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python Environment
+        uses: actions/setup-python@v2.2.2
```

```suggestion
        uses: actions/setup-python@v2.2
```
I wouldn't pin to a patch.

---

I can just pin to a major version as there shouldn't be any breaking changes.

---

Sounds good.